src/parser/av1_parser.h

```diff
@@ -56,21 +56,29 @@ class Av1VideoParser : public RocVideoParser {
     virtual rocDecStatus UnInitialize();     // derived method
 
     typedef struct {
-        int index;
-    } Av1Picture;
-
-    typedef struct {
-        uint32_t offset;
-        uint32_t size;
+        uint32_t tile_offset;
```

Comment:
do you need "tile_" in member names? It is implied

Reply:
It is for naming consistency with other parameters like tile_col and tile_row.